src/pages/Login/Login.js

```diff
@@ -1,7 +1,105 @@
-import React from 'react';
+import React, { useState } from 'react';
+import styled, { css } from 'styled-components';
+import { FaPinterest } from 'react-icons/fa';
+import LoginModal from './LoginModal';
```

Comment:
import 순서 수정해주세요! 일반적인 convention을 따라 순서만 잘 지켜주셔도 가독성이 좋아집니다. 아래 순서 참고해주세요.

- React → Library(Package) → Component → 변수 / 이미지 → css 파일(scss 파일)
```suggestion
import React, { useState } from 'react';
import { FaPinterest } from 'react-icons/fa';
import LoginModal from './LoginModal';
import styled, { css } from 'styled-components';
```

Reply:
수정했습니다. 다음 commit에서는 잘 지키도록 하겠습니다!!